<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGB_Viewer</class>
 <widget class="QMainWindow" name="RGB_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="image_box">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>rgb image preview</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="image_layout"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>rgb image parameter</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Image Size :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QComboBox" name="size_combox">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <item>
               <property name="text">
                <string>&lt;Custom Size&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>VGA (640x480)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>WVGA (832x480)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ITU (720x576)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>720P (1280x720)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1080P (1920x1080)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4K (3840x2160)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>XGA (1024x768)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>XGA+ (1280x960)</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Image Width :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="imwidth">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Image Height :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QLineEdit" name="imheight">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Sensor Bits : </string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QSpinBox" name="spin_sensorbits">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="minimum">
               <number>8</number>
              </property>
              <property name="maximum">
               <number>16</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>12</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="btn_open">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="save">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>148</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
